js_modules/dagit/packages/core/src/workspace/VirtualizedJobTable.tsx

```diff
@@ -172,7 +172,9 @@ const JobRow = (props: JobRowProps) => {
         </RowCell>
         <RowCell>
           {latestRuns.length ? (
-            <RunStatusPezList jobName={name} runs={[...latestRuns].reverse()} fade />
+            <Box margin={{top: 4}}>
```

Comment:
If the goal is to center this, I wonder if it'd be more fool-proof to do a `RowCell style={{justifySelf: 'center'}}` to override the flexbox default?

Reply:
I don't really want to center it exactly, I want to align it a little bit better with the other cells that now have top-aligned content.